doc/manuals/user/transient_entities.md

```diff
@@ -0,0 +1,124 @@
+# Transient Entities
+
+Orion Context Broker implements *transient entities*. A transient entity is like any other entity 
+(i.e. it has id/type, a set of attributes, etc.) but with a limited lifetime, so when a given
+time has come, the entity is automatically deleted from the context managed by Orion.
+
+Thus, a first and very important piece of advice: **be careful if you use transient entities as once
+the expiration time has come, the entity will be automatically deleted from database and there is
+no way of recovering it**. Ensure the information you set in a transient entity is actually "transient"
```

Comment:
it is not relevant once the Entity has expired (i.e. deleted)

Reply:
Fixed in Fixed in 78286a073f3b775bed26ebf74bb17da2bea33fe9